File: TeamCode/src/main/java/linearOpModes/TestOpMode.java
package linearOpModes;

import com.qualcomm.robotcore.eventloop.opmode.Disabled;
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
import com.qualcomm.robotcore.eventloop.opmode.TeleOp;

import org.woen.telemetry.ThreadedTelemetry;
import org.woen.utils.units.Vec2;

import neural.NeuralNetwork;

@Disabled
@TeleOp
public class TestOpMode extends LinearOpMode {
    @Override
    public void runOpMode() throws InterruptedException {
        waitForStart();
        resetRuntime();

        while (opModeIsActive())
        {
            NeuralNetwork test = new NeuralNetwork();

            test.SaveParameters();

            sleep(1000);
        }
    }
}

File: TeamCode/src/main/kotlin/org/woen/linearOpModes/TestOpMode.kt
package org.woen.linearOpModes

import com.acmerobotics.dashboard.FtcDashboard
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode
import com.qualcomm.robotcore.eventloop.opmode.TeleOp

@TeleOp
class TestOpMode: LinearOpMode() {
    override fun runOpMode() {
        waitForStart()
        resetRuntime()

        while (opModeIsActive()){
            sleep(20)

            FtcDashboard.getInstance().telemetry.addData("a", System.currentTimeMillis())

            FtcDashboard.getInstance().telemetry.update()
        }
    }
}